memstore/memstore.go

```diff
@@ -264,7 +264,7 @@ func (m *memStoreImpl) AddTableShard(table string, shardID int, needPeerCopy boo
 	defer m.Unlock()
 
 	schema := m.TableSchemas[table]
-	if schema != nil {
+	if schema == nil {
```

Comment:
error handling or output some error msg?

Reply:
this is not really an error